controllers/providers/kubernetes/utils.go

```diff
@@ -254,6 +254,11 @@ func ParseCustomResourceYaml(raw string) (*unstructured.Unstructured, error) {
 
 func ConfigmapHash(cm *corev1.ConfigMap) string {
 	var buf strings.Builder
+	defer func() {
```

Comment:
Did you see this function getting a panic?

Reply:
yeah, meant to not include this - it was causing panics running in GoLand debug mode - but not outside of that.